src/backend/distributed/planner/multi_logical_optimizer.c

```diff
@@ -1168,9 +1181,12 @@ TransformSubqueryNode(MultiTable *subqueryNode, List *tableNodeList)
 	MultiNode *collectChildNode = ChildNode((MultiUnaryNode *) collectNode);
 	bool groupedByDisjointPartitionColumn =
 		GroupedByDisjointPartitionColumn(tableNodeList, extendedOpNode);
-	MultiExtendedOp *masterExtendedOpNode = MasterExtendedOpNode(extendedOpNode);
+	MultiExtendedOp *masterExtendedOpNode = MasterExtendedOpNode(extendedOpNode,
```

Comment:
Minor visual. It would be better if we start `MasterExtendedOpNode` on the next line.

Reply:
ok